### ISSUE TITLE
code blocks comments fail lighthouse color contrast check in dark mode

### ISSUE THREAD

**Issue by hichemfantar** [shuding/nextra#4069]
lighthouse accessibility check

![Image](https://github.com/user-attachments/assets/703b6777-b31d-4b7d-98d7-26a4510a46e2)
